site/pages/view-all-disruptions.page.tsx

```diff
@@ -118,10 +119,22 @@ const formatDisruptionsIntoRows = (disruptions: TableDisruption[], offset: numbe
             header: (
                 <Link
                     className="govuk-link"
-                    href={{
-                        pathname: `/disruption-detail/${disruption.id}`,
-                        query: { return: VIEW_ALL_DISRUPTIONS_PAGE_PATH },
-                    }}
+                    href={
+                        disruption.status === Progress.draft
+                            ? disruption.consequenceLength && disruption.consequenceLength > 0
+                                ? {
+                                      pathname: `/review-disruption/${disruption.id}`,
```

Comment:
Can we use the constants for these page paths?

Reply:
done